RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -7,12 +7,168 @@
+            trackSlider
+            HStack {
+                backwardButton
+                playPauseButton
+                stopButton
+                forwardButton
+            }
+            downloadButton
+            shareButton
+        }.accentColor(.red)
+    }
+    
+    // MARK: - Private Properties
+    
+    private var posterImage: some View {
+        Image(player.currentTrack?.poster ?? Constants.emptyString)
+            .resizable()
+            .scaledToFit()
+            .frame(width: Constants.posterWidthHeight,
+                   height: Constants.posterWidthHeight)
+            .background(Color.orange)
+            .cornerRadius(Constants.posterCornerRadius)
+            .shadow(color: .red,
+                    radius: Constants.shadowRadius,
+                    x: Constants.shadowX,
+                    y: Constants.shadowY)
+    }
+    
+    private var trackSlider: some View {
+        Slider(value: Binding(get: {
+            player.trackCurrentTime ?? 0
+        }, set: { newValue in
+            progress = newValue
+            player.setTrackTime(value: newValue)
+        }), in: 0...Float(player.maxDuration)) {
+        } minimumValueLabel: {
+            Text(Constants.emptyString)
+        } maximumValueLabel: {
+            Text("\(player.remainingTimeName)")
         }
         .padding()
+        .accentColor(.red)
+    }
+    
+    private var backwardButton: some View {
+        Button {
+            player.rewind()
+        } label: {
+            Image(systemName: Constants.rewindButtonImageName)
+        }.frame(width: Constants.buttonLoWidth,
+                height: Constants.buttonHeight)
+            .background(Color.orange)
+            .cornerRadius(Constants.buttonCornerRadius)
+    }
+    
+    private var playPauseButton: some View {
+        Button {
+            player.playPause()
+        } label: {
+            Text(player.isPlaying ? Constants.pauseButtonName : Constants.playButtonName)
+        }
+        .frame(width: Constants.buttonHiWidth,
+               height: Constants.buttonHeight)
+        .background(Color.orange)
+        .cornerRadius(Constants.buttonCornerRadius)
+    }
+    
+    private var stopButton: some View {
+        Button {
+            player.stop()
+        } label: {
+            Text(Constants.stopButtonName)
+        }
+        .frame(width: Constants.buttonHiWidth,
+               height: Constants.buttonHeight)
+        .background(Color.orange)
+        .cornerRadius(Constants.buttonCornerRadius)
+    }
+    
+    private var forwardButton: some View {
+        Button {
+            player.forward()
+        } label: {
+            Image(systemName: Constants.forwardButtonImageName)
+        }.frame(width: Constants.buttonLoWidth,
+                height: Constants.buttonHeight)
+            .background(Color.orange)
+            .cornerRadius(Constants.buttonCornerRadius)
+    }
+    
+    private var downloadButton: some View {
```

Comment:
```suggestion
    private var downloadButton: some View {
```
У меня был коммент что если эта кнопка что-то показывает то надо указывать Shown, но это не точно